app/src/test/java/tech/relaycorp/gateway/ExampleUnitTest.kt

```diff
@@ -4,8 +4,10 @@ import org.junit.jupiter.api.Assertions.assertEquals
 import org.junit.jupiter.api.Test
 
 class ExampleUnitTest {
+    private val very_looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong_variable = 4
+
     @Test
     internal fun test() {
-        assertEquals(4, 2 + 2)
+        assertEquals(very_looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong_variable, 1)
```

Comment:
This test is broken, but `gradle test` doesn't spot it because it's not running the test suite.